modules/ROOT/pages/indexes/search-performance-indexes/create-indexes.adoc

```diff
@@ -0,0 +1,762 @@
+:description: This page explains how to create indexes used for search performance.
+= Create indexes
+
+Creating an index is done with the `CREATE [index_type] INDEX [index_name]` command.
+If no index type is specified in the create command a xref:indexes/search-performance-indexes/create-indexes.adoc#create-range-index[range index] is created.
```

Comment:
wouldn't it be nice to still list which kinds of indexes this page covers? so reformulating this instead of just removing it?
```
The following index types are included in this category:

* xref:indexes/search-performance-indexes/managing-indexes.adoc#create-range-index[Range indexes]
* xref:indexes/search-performance-indexes/managing-indexes.adoc#create-text-index[Text indexes]
* xref:indexes/search-performance-indexes/managing-indexes.adoc#create-point-index[Point indexes]
* xref:indexes/search-performance-indexes/managing-indexes.adoc#create-lookup-index[Token lookup indexes]
```